client/my-sites/plans/current-plan/jetpack-checklist/constants.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Returns the localized duration of a task in given minutes.
+ *
+ * @param  {Number} minutes Number of minutes.
+ * @return {String} Localized duration.
+ */
+export function getJetpackChecklistTaskDuration( minutes ) {
+	return translate( '%d minute', '%d minutes', { count: minutes, args: [ minutes ] } );
+}
+
+export const JETPACK_CHECKLIST_TASKS = {
+	jetpack_backups: {
+		title: translate( 'Backups & Scanning' ),
+		description: translate(
+			"Connect your site's server to Jetpack to perform backups, rewinds, and security scans."
+		),
+		completedButtonText: translate( 'Change' ),
+		completedTitle: translate( 'You turned on backups and scanning.' ),
+		getUrl: siteSlug => `/activity-log/${ siteSlug }`,
+		duration: getJetpackChecklistTaskDuration( 2 ),
+	},
+	jetpack_monitor: {
+		title: translate( 'Jetpack Monitor' ),
```

Comment:
I think this "brand name" is no more and we call it now "Downtime monitoring" https://jetpack.com/support/monitor/

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/31547/commits/0c11570c987b23704890c17962bf75fc05c5425b